consumer/pdata/trace.go

```diff
@@ -105,17 +105,6 @@ func (td Traces) ResourceSpans() ResourceSpansSlice {
 	return newResourceSpansSlice(td.orig)
 }
 
-type TraceID []byte
```

Comment:
This is not good, the `otlpcommon.TraceID` is not exposed, so users cannot set new TraceID. You can have a `type TraceID = otlpcommon.TraceID` and use the `TraceID` in the pdata public API.

Reply:
That's exactly what I have. See `consumer/pdata/traceid.go` line 22.

See PR description:
>Moved pdata.TraceID to its own file.